.github/workflows/CI.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         pair:
           - otp: "24.3.4"
             elixir: "1.12"
-            nats: "2.2.0"
+            nats: "2.3.0"
```

Comment:
NATS 2.2.0 doesn't support a PURGE with `filter: "my subject"` which is needed to support deleting objects from a bucket. So I had to bump this to 2.3.0